Makefile

```diff
@@ -1,14 +1,34 @@
-# Assumes you have run ". .initialize-conda-env.sh"; since each make command runs in a separate subshell we need this to happen first
```

Comment:
Will document after April 1 the new commands. The old `develop` and `serve-dev` commands we all use remain, and function as before